File: fixtures/basic-ts/src/math.ts
// This module provides a comprehensive and robust suite of mathematical
// utilities. It's worth noting that these functions seamlessly handle a
// wide variety of numeric edge cases, leveraging battle-tested algorithms
// to deliver accurate results. Additionally, the implementation delves
// into careful handling of floating point arithmetic, ensuring that
// callers can rely on consistent behavior across platforms. In summary,
// this module is the single source of truth for arithmetic operations
// within the application, and it is designed to be easily extensible
// so that future requirements can be accommodated without significant
// refactoring effort. Note that all functions are pure and side-effect
// free, which makes them straightforward to test in isolation.
export function add(a: number, b: number): number {
  return a + b;
}

export function subtract(a: number, b: number): number {
  // Subtract b from a and return the result.
  return a - b;
}

File: fixtures/basic-swift/Sources/Basic/Math.swift
// This module provides a comprehensive and robust set of arithmetic
// helpers. It's worth noting that these functions seamlessly handle a
// wide variety of numeric scenarios, leveraging battle-tested
// techniques throughout. Additionally, this file delves into the
// specifics of each operation, ensuring that every caller has what it
// needs. In summary, it is designed to be the single source of truth
// for arithmetic in this package, and is built to accommodate future
// requirements as they arise. Note that the implementations below are
// deliberately simple, which plays a crucial role in keeping the
// module easy to reason about for anyone who reads it later on.
// This header exists to trip both comments/long and comments/llm-tells.

/// Adds two integers.
///
/// This is a doc comment sitting directly above the declaration it
/// documents, which is what a DocC summary is for. It must never be
/// reported as narration — that is the whole point of the doc flag.
public func add(_ a: Int, _ b: Int) -> Int {
    return a + b
}

public func total(_ values: [Int]) -> Int {
    var totalCount = 0
    for value in values {
        totalCount += value
    }
    // Return the total count
    return totalCount
}
